File: lib_local/gallery/src/main/java/com/yanzhenjie/album/impl/OnItemClickListener.java
package com.yanzhenjie.album.impl;

import android.view.View;

/**
 * <p>Listens on the item's click.</p>
 * Created by Yan Zhenjie on 2016/9/23.
 */
public interface OnItemClickListener {

    /**
     * When Item is clicked.
     *
     * @param view     item view.
     * @param position item position.
     */
    void onItemClick(View view, int position);

}

File: lib_local/gallery/src/main/java/com/example/gallery/album/widget/recyclerview/OnItemClickListener.kt
package com.example.gallery.album.widget.recyclerview

import android.view.View

/**
 * 列表条目点击事件监听器
 * 用于监听 RecyclerView / ListView 条目的点击回调
 */
interface OnItemClickListener {
    /**
     * 当条目被点击时回调
     * @param view     被点击的条目视图
     * @param position 被点击的条目位置
     */
    fun onItemClick(view: View?, position: Int)
}